### DISCUSSION TITLE
How to configure Prometheus monitoring for BookKeeper?

### DISCUSSION THREAD

**Discussion by yunyaoqiao** [milvus-io/milvus#28959] [Q&A and General discussion]
http://milvus-dev-pulsar.xxx.net:8000/metrics returns Bad Request. and the error return is:
{"statusCode":404,"message":"Error: ENOENT: no such file or directory, stat '/app/dist/build/index.html'","error":"Bad Request"}

i has changed bookkeeper.conf, which prometheusStatsHttpPort=8000, httpServerEnabled=true, httpServerPort=8100.  but still can't get metrics

**Comment by yhmo** [comment]
I don't think it is a milvus issue. You can browse the pulsar document for more information: https://pulsar.apache.org/docs/2.10.x/deploy-monitoring/
